### PR TITLE
Parametrise Debian image builder so I don't have to build everything all the time.

### PR DESCRIPTION
**Pull request by tomwilkie**
Eg:

```
$ docker run --volume="$(pwd)/debian:/src" debian-packager -arch=arm -distros=jessie
```

### CODE REVIEW

**Comment by tomwilkie** [comment]
I signed it!

File: debian/build.go [hunk 1]
```diff
@@ -60,6 +70,12 @@ var (
 	keepTmp = flag.Bool("keep_tmp", false, "keep tmp dir after build")
 )
 
+func init() {
+	flag.Var(&architectures, "arch", "Architectures to build for.")
+	flag.Var(&serverDistros, "server_distros", "Server distros to build for.")
```
Comment (ixdy) [inline debian/build.go]: maybe use a dash instead of underscore to keep consistency with the main repo?

**Comment by tomwilkie** [comment]
Sure!  Want me to change `keep_tmp` as well?

**Comment by ixdy** [comment]
sure

**Comment by tomwilkie** [comment]
@ixdy PTAL

**Pull request merged**